README.md

```diff
@@ -64,23 +64,32 @@ This project can be run in a [Docker](https://www.docker.com/) container.
 You need a local database instance listening on port 8080. Here is how to set that up:
 
 ``` sh
-docker run --name mysql -e MYSQL_ROOT_PASSWORD=password -p 3306:3306 -d mysql
-export LEDGER_DB_URI=mysql://root:password@localhost/fivebells
-npm run migrate
+docker run --name db -e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres
+export LEDGER_DB_URI=postgres://postgres:password@localhost
```

Comment:
good point as well, and actually turns out it's not needed to expose the postgres db port to the host because we use container linking